task/sql-tasks.js

```diff
@@ -199,7 +310,24 @@ async function task_1_14(db) {
  *
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    select 
+        sum(month(OrderDate) = 1)  as "January",

```

Comment:
use count

Reply:
but if i'll use count i'll get the quantity all records